packages/core/src/lib/uikitExcludePattern.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 
+var _ = require('lodash');
```

Comment:
Eslint will throw an error on this line because var is not allowed.

```suggestion
const _ = require('lodash');
```